tests/README.md

```diff
@@ -77,9 +82,14 @@ To run integration tests use the "integration_tests.sh" test run script.
 ./integration.sh
 ```
 
-**Example: Run with alternate PE package repo on Windows 2008 R2**
+**Example: Run with alternate PE package repo on Windows 2008 R2 using Forge**
+```
+./integration.sh windows-2008r2-64mda http://alt.address.local/3.8.2/preview forge
+```
+
+**Example: Run with alternate PE package repo on Windows 2012 R2 with local module code (No Forge)**
 ```
-./integration.sh windows-2008r2-64mda http://alt.address.local/3.8.2/preview
+./acceptance.sh windows-2012r2-64a http://alt.address.local/3.8.2/preview local
```

Comment:
And also noticed that these should be `acceptance_tests` and `integration_tests` now as well.  Fixing up in my local testing.